Makefile

```diff
@@ -54,6 +54,9 @@ init: node_modules ;
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
```

Comment:
I would actually suggest _not_ having as part of the default run. I like to keep the default target as lean as passible, sort of a minimum run to get everything in place of of the box. Extra things you'd use during a development cycle but that might not be needed in all cases shouldn't be part of this.

Note for example we're not running `test` as part of `default`. It should of course be run after making code changes but you might not want it at every step as you work through something. Having `lint` as part of the `ci` target is also a good thing.

An `all` target separate from `default` could include a heavier load.